src/main/java/net/sf/jabref/collab/ChangeScanner.java

```diff
@@ -165,37 +165,16 @@ private void storeTempDatabase() {
         });
     }
 
-    private void scanMetaData(MetaData inMem1, MetaData inTemp1, MetaData onDisk) {
-        MetaDataChange metadataChange = new MetaDataChange(inMem1, inTemp1);
-        List<String> handledOnDisk = new ArrayList<>();
-        // Loop through the metadata entries of the "tmp" database, looking for
-        // matches
-        for (String key : inTemp1) {
-            // See if the key is missing in the disk database:
-            List<String> vod = onDisk.getData(key);
-            if (vod == null) {
-                metadataChange.insertMetaDataRemoval(key);
-            } else {
-                // Both exist. Check if they are different:
-                List<String> vit = inTemp1.getData(key);
-                if (!vod.equals(vit)) {
-                    metadataChange.insertMetaDataChange(key, vod);
-                }
-                // Remember that we've handled this one:
-                handledOnDisk.add(key);
+    private void scanMetaData(MetaData inMemory, MetaData onTmp, MetaData onDisk) {
```

Comment:
Yes the detailed information about which key has changed is not contained in the event. Of course this information could be easily extracted from the two metadata by comparing each contained meta item separately. I didn't found this information to useful and thus just left a todo https://github.com/JabRef/jabref/pull/2112/files#diff-423409b75a40e426ad5123cabbe295e6R33.